localstack-core/localstack/config.py

```diff
@@ -1071,6 +1071,24 @@ def populate_edge_configuration(
 # DEV: sbx_user1051 (default when not provided) Alternative system user or empty string to skip dropping privileges.
 LAMBDA_INIT_USER = os.environ.get("LAMBDA_INIT_USER")
 
+# INTERNAL: 1 (default)
+# The duration (in seconds) to wait between each poll call to an event source.
+LAMBDA_EVENT_SOURCE_MAPPING_POLL_INTERVAL_SEC = float(
+    os.environ.get("LAMBDA_EVENT_SOURCE_MAPPING_POLL_INTERVAL_SEC") or 1
+)
+
+# INTERNAL: 60 (default)
+# Maximum duration (in seconds) to wait between retries when an event source poll fails.
+LAMBDA_EVENT_SOURCE_MAPPING_MAX_BACKOFF_ON_ERROR_SEC = float(
```

Comment:
Thanks for marking this as internal 👍 

I understand that it can be helpful for quick experimentation with some customers, but we should be generally cautious making everything configurable, which introduces extra complexity. These are candidates to remove if there is no strong need in favor of choosing sensible defaults.